boards/st/stm32f469i_disco/stm32f469i_disco.dts

```diff
@@ -138,3 +170,102 @@ zephyr_udc0: &usbotg_fs {
 	pinctrl-names = "default";
 	cd-gpios = <&gpiog 2 GPIO_ACTIVE_LOW>;
 };
+
+&sdram1 {
+	/* Frame buffer memory cache will cause screen flickering. */
+	zephyr,memory-attr = <( DT_MEM_ARM(ATTR_MPU_RAM_NOCACHE) )>;
+};
+
+&ltdc {
+	status = "okay";
+	ext-sdram = <&sdram1>;
+	width = <800>;
+	height = <480>;
+	def-back-color-red = <0>;
+	def-back-color-green = <0>;
+	def-back-color-blue = <0>;
+	pixel-format = <PANEL_PIXEL_FORMAT_RGB_565>;
+
+	display-timings {
+		compatible = "zephyr,panel-timing";
+		hsync-active = <0>;
+		vsync-active = <0>;
+		de-active = <0>;
+		pixelclk-active = <0>;
+		hsync-len = <2>;
+		vsync-len = <120>;
+		hback-porch = <34>;
+		vback-porch = <150>;
+		hfront-porch = <34>;
+		vfront-porch = <150>;
+	};
+};
+
+&mipi_dsi {
+	status = "okay";
+	pll-ndiv = <125>;
+	pll-idf = <2>;
+	pll-odf = <0>;
+	vs-active-high;
+	hs-active-high;
+	de-active-high;
+
+	loosely-packed;
+	largest-packet-size = <64>;
+	bta-ack-disable;
+	lp-rx-filter = <10000>;
+
+	nt35510: nt35510@0 {
+		status = "okay";
+		compatible = "frida,nt35510";
+		reg = <0x0>;
+		height = <800>;
+		width = <480>;
+		reset-gpios = <&gpioh 7 GPIO_ACTIVE_HIGH>;
+		bl-gpios = <&gpioa 3 GPIO_ACTIVE_HIGH>;
+		data-lanes = <2>;
+		pixel-format = <PANEL_PIXEL_FORMAT_RGB_565>;
+		rotation = <90>;
+	};
```

Comment:
This is already defined in `boards/shields/st_b_lcd40_dsi1_mb1166/st_b_lcd40_dsi1_mb1166_a09.overlay`

Reply:
See https://github.com/zephyrproject-rtos/zephyr/pull/74646 for how to add support for this board in the MB1166 shield's folder.